kratos/python_scripts/json_output_process.py

```diff
@@ -104,14 +104,14 @@ def ExecuteBeforeSolutionLoop(self):
         data["TIME"] = []
         count = 0
         for node in self.sub_model_part.Nodes:
-
             compute = self.__check_flag(node)
 
             if (compute == True):
                 if (self.resultant_solution == False):
                     data["NODE_" + str(node.Id)] = {}
                 else:
-                    data["RESULTANT"] = {}
+                    if (count == 0):
```

Comment:
@loumalouomega you are the master here... can you take a look?

Reply:
OK, I will cherry pick and isolate the changes

Reply:
ping @loumalouomega ? is this fine to you?

Reply:
ping @loumalouomega